resources/testdriver.js

```diff
@@ -137,27 +137,64 @@
         }
     };
 
-    window.test_driver_internal = {
+    var manual = {
         /**
-         * Triggers a user-initiated click
+         * This flag should be set to `true` by any code which implements the
+         * internal methods defined below for automation purposes. Doing so
+         * allows the library to signal failure immediately when an automated
+         * implementation of one of the methods is not available.
+         */
+        in_automation: false,
+
+        /**
+         * Waits for a user-initiated click
          *
          * @param {Element} element - element to be clicked
          * @param {{x: number, y: number} coords - viewport coordinates to click at
-         * @returns {Promise} fulfilled after click occurs or rejected if click fails
+         * @returns {Promise} fulfilled after click occurs
          */
         click: function(element, coords) {
-            return Promise.reject(new Error("unimplemented"));
+            if (window.test_driver_internal.in_automation) {
```

Comment:
```suggestion
            if (this.in_automation) {
```

Reply:
This relates to the change to use a prototype. Do we treat `test_driver_internal` as a namespace like `console`, or an instance with a prototype like `document`? If you revert the `Object.create` change, please also reject these suggestions.